benchmarks/helics/PholdFederate.hpp

```diff
@@ -47,18 +47,27 @@ class PholdFederate {
     std::uniform_real_distribution<double> rand_uniform_double;
     std::uniform_int_distribution<unsigned int> rand_uniform_int;
 
+    // callbacks for instrumenting the code
```

Comment:
Could you envision doing something like this for some of the other benchmarks, potentially making a base class or something of that nature?

Reply:
Yes, there's parts that would make sense to pull out into a base class that could provide default callbacks and command line arguments.